.github/workflows/operator-integration-test.yml

```diff
@@ -110,3 +110,202 @@ jobs:
           cd integration-tests/manifests/cmd
           go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json
           kubectl delete instrumentation sample-instrumentation
+
+
+
+  DeploymentAnnotationsTest:
```

Comment:
non blocking nit: now that we have these annotations test, we can change the original integration test job to be called "InstrumentationTest" or something.